installation/install-by-deb.md

```diff
@@ -90,7 +90,7 @@ $ sudo systemctl status td-agent.service
 ● td-agent.service - td-agent: Fluentd based data collector for Treasure Data
    Loaded: loaded (/lib/systemd/system/td-agent.service; disabled; vendor preset: enabled)
    Active: active (running) since Thu 2017-12-07 15:12:27 PST; 6min ago
-     Docs: https://docs.treasuredata.com/articles/td-agent
+     Docs: https://docs.treasuredata.com/display/public/PD/Installing+td-agent+on+Ubuntu+and+Debian
```

Comment:
Thanks for notifying it, but the URL is inappropriate since it's an example of command line output of `systemctl status`.
For td-agent v3, it was https://docs.treasuredata.com/articles/td-agent in actual.
For td-agent v4, we show `https://docs.treasuredata.com/display/public/PD/About+Treasure+Data%27s+Server-Side+Agent` for this, not `https://docs.treasuredata.com/display/public/PD/Installing+td-agent+on+Ubuntu+and+Debian`. Here is the actual output of `systemctl status` with td-agent v4:
```
● td-agent.service - td-agent: Fluentd based data collector for Treasure Data
     Loaded: loaded (/lib/systemd/system/td-agent.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2022-09-20 05:28:25 JST; 1 week 0 days ago
       Docs: https://docs.treasuredata.com/display/public/PD/About+Treasure+Data%27s+Server-Side+Agent
   Main PID: 2417 (fluentd)
      Tasks: 8 (limit: 38328)
     Memory: 52.2M
        CPU: 56.868s
     CGroup: /system.slice/td-agent.service
             ├─2417 /opt/td-agent/bin/ruby /opt/td-agent/bin/fluentd --log /var/log/td-agent/td-agent.log --daemon /var/run/td-agent/td-agent.pid
             └─2420 /opt/td-agent/bin/ruby -Eascii-8bit:ascii-8bit /opt/td-agent/bin/fluentd --log /var/log/td-agent/td-agent.log --daemon /var/run/td-agent/t>

 9月 20 05:28:24 Ryzen systemd[1]: Starting td-agent: Fluentd based data collector for Treasure Data...
 9月 20 05:28:25 Ryzen systemd[1]: Started td-agent: Fluentd based data collector for Treasure Data.
```

Since some other values are also changed in the output, we should replace the example entirely.